src/Tizen.NUI/src/internal/BackKeyManager.cs

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright(c) 2019 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// BackKeyManager is a class to manager back key event.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class BackKeyManager
+    {
+
+        /// <summary>
+        /// BackKeyManager construct.
+        /// </summary>
+        private BackKeyManager()
+        {
+            NUIApplication.GetDefaultWindow().KeyEvent += OnWindowKeyEvent;
+        }
+
+        /// <summary>
+        /// Subscriber.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<View> Subscriber{get; set;} = new List<View>();
+
+        /// <summary>
+        /// BackKeyManager static instance.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static BackKeyManager Instance { get; } = new BackKeyManager();
```

Comment:
It seems it could get multiple instances when BackKeyManager.Instance is called multiple times. I am not sure so could you check the FocusManager singleton pattern? ^^;

https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/FocusManager.cs#L276

Reply:
I used Lazy<T> pattern but it works when using .NET4 or higher.

https://csharpindepth.com/articles/singleton

Do you think we can use this?

Reply:
DONE

Reply:
great! (b^_^)b
sorry but I couldn't see Lazy<T> and I think FocusManager uses "forth version" of the article above.
if you use Instance by new BackKeyManager, null checking need be added, I think.
in my private opinion, following FocusManger pattern would be good.
How do you think about?

Reply:
Follow FocusManager.
Please check changes.